chia/wallet/wallet_state_manager.py

```diff
@@ -292,44 +292,50 @@ async def create_more_puzzle_hashes(self, from_zero: bool = False, in_transactio
             # If the key was replaced (from_zero=True), we should generate the puzzle hashes for the new key
             if from_zero:
                 start_index = 0
+            last_index = unused + to_generate
+            if start_index >= last_index:
+                self.log.debug(f"Nothing to scan for for wallet_id: {wallet_id}, index: {start_index}")
+            else:
+                scanning_msg = f"Scanning puzzle hashes from {start_index} to {last_index} for wallet_id: {wallet_id}"
```

Comment:
I would call it "Creating puzzle hashes" instead of "Scanning puzzle hashes", since it's not really a scan

Reply:
Makes sense, see [1781682](https://github.com/Chia-Network/chia-blockchain/pull/10761/commits/1781682f628617c8d442eeb54a67e6cdab3d1fd7)